crawl-ref/docs/options_guide.txt

```diff
@@ -117,7 +117,7 @@ The contents of this text are:
                 tile_tag_pref, tile_window_width, tile_window_height,
                 tile_map_pixels, tile_viewport_scale, tile_map_scale,
                 tile_cell_pixels, tile_sidebar_pixels, tile_filter_scaling,
-                tile_force_overlay, tile_overlay_col,
+                tile_force_overlay, tile_overlay_col, min_stat_width_characters,
```

Comment:
This new option works great, thank you. But isn't there a small typo here? I mean it's missing the tile prefix.